src/screens/map-screen/map-screen.component.tsx

```diff
@@ -300,11 +317,13 @@ const MapScreen = () => {
           travelState={travelState}
         />
 
-        <BuildingInformation
-          tappedBuilding={tappedBuilding}
-          showBuildingInfo={showBuildingInfo}
-          onClosePanel={onClosePanel}
-        />
+        {travelState === TravelState.NONE && (
```

Comment:
Use the ternary operator for consistency. Sometimes the linter complaints when using the && for conditionals. That's not the case here (since travis passed), but since we've been doing ternary up until now, it would be best to be consistent.

Reply:
In this file && is used so we followed it for consistency. Should we change all of it?

Reply:
If that's the case then nevermind, I didn't notice.